new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Operator.java

```diff
@@ -3,35 +3,78 @@
 
 package oracle.weblogic.kubernetes.actions.impl;
 
+import io.kubernetes.client.openapi.ApiException;
 import oracle.weblogic.kubernetes.actions.impl.primitive.Helm;
+import oracle.weblogic.kubernetes.extensions.LoggedTest;
 
-public class Operator {
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class Operator implements LoggedTest {
 
   /**
    * The URL of the Operator's Helm Repository.
    */
   private static String OPERATOR_HELM_REPO_URL = "https://oracle.github.io/weblogic-kubernetes-operator/charts";
+
   /**
    * The name of the Operator's Helm Chart (in the repository).
    */
   private static String OPERATOR_CHART_NAME = "weblogic-operator/weblogic-operator";
 
+  /**
+   * Install WebLogic Kubernetes Operator.
+   *
+   * @param params operator parameters for helm values
+   * @return true if the operator is successfully installed, false otherwise.
+   * @throws ApiException - if Kubernetes client API call fails
+   */
+  public static boolean install(OperatorParams params) throws ApiException {
+
+    String namespace = params.getNamespace();
+    String serviceAccount = params.getServiceAccount();
+
+    // assertions for required parameters
+    assertNotNull(namespace, "Operator namespace cannot be null");
```

Comment:
use assertj - will make this look like
```   
import static org.assertj.core.api.Assertions.*;

 assertThat(namespace)
            .as("make sure namespace is not empty or null")
            .isNotEqualTo("")
            .isNotNull();
```
which is a lot clearer

Reply:
looking for something similar to isNotNullOrEmpty() as it takes care of multiple white spaces..

Reply:
We have Guava as a dependency in other modules.  You could add it here and use: assertFalse(Strings.isNullOrEmpty(text));

Reply:
added assertj core dependency and doing ..
```
    assertThat(namespace)
        .as("make sure namespace is not empty or null")
        .isNotNull()
        .isNotEmpty();
```